### DISCUSSION TITLE
Update titiler-core on conda-forge channel

### DISCUSSION THREAD

**Discussion by nmcdtu** [developmentseed/titiler#950] [Q&A] [answered]
The current version of `titiler-core` on `conda-forge` is 0.17.0.
https://anaconda.org/conda-forge/titiler-core

May you consider updating it to the latest version?
Many thanks!

**Comment by vincentsarago** [comment, marked as answer]
@nmcdtu we are not the maintainers of the Conda repo. you may ask over https://github.com/conda-forge/titiler-core-feedstock
